middleware/classes.py

```diff
@@ -130,3 +131,18 @@ def process_request(self, request):
             django_timezone = None
         if django_timezone:
             timezone.activate(django_timezone)
+
+
+class RaiseResponse:
```

Comment:
I feel like we could come up with a better name

Reply:
Changed name to `CatchRaisedExceptionResponseMiddleware`